ValueError: No constraint named unique_user_email on model User

Hi, even after following the various instructions for installing the app, I receive the following error when applying migrations.
```
 File "~/.local/lib/python3.11/site-packages/django/db/migrations/state.py", line 968, in get_constraint_by_name
    raise ValueError("No constraint named %s on model %s" % (name, self.name))
ValueError: No constraint named unique_user_email on model User
make: *** [Makefile:59: migrate] Error 1
```
python version 3.11
django version Django==4.2.7
postgresql version 14

--- a/src/unique_user_email/apps.py
+++ b/src/unique_user_email/apps.py
@@ -23,4 +23,4 @@ class UniqueUserEmailConfig(AppConfig):
         ]
         User._meta.constraints = User.Meta.constraints
         # ... as long as original_attrs is not updated.
-        # User._meta.original_attrs["constraints"] = User.Meta.constraints
+        User._meta.original_attrs["constraints"] = User.Meta.constraints

--- a/tests/tests.py
+++ b/tests/tests.py
@@ -6,6 +6,7 @@ from django.db import IntegrityError
 from django.test import TestCase
 from unique_user_email.backend import EmailBackend
 from unique_user_email.forms import AuthenticationForm
+from django.db.models.constraints import UniqueConstraint
 
 
 class UniqueEmailTestCase(TestCase):
@@ -55,6 +56,13 @@ class UniqueEmailTestCase(TestCase):
         ):
             user2.save()
 
+    def test_user_constraints(self):
+        self.assertIsInstance(User._meta.constraints[0], UniqueConstraint)
+        self.assertEqual("unique_user_email", User._meta.constraints[0].name)
+        self.assertEqual(
+            "unique_user_email",
+            User._meta.original_attrs.get("constraints")[0].name
+        )
 
 class EmailBackendTests(TestCase):
     def test_none_for_username_logins(self):

